### PR TITLE
OracleCorporation.MySQLWorkbench8.0CE version 8.0.28

### PR DESCRIPTION
**Pull request by filipponova**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/41333)

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
image

**Comment by filipponova** [comment]
> image



> image

Thank you for your help. I was looking into the problem, can I add dependencies in my manifest or is this still not supported? Sorry for the question, it's my first PR here, I'm still reading the docs.

File: manifests/o/OracleCorporation/MySQLWorkbench8/0CE/8.0.28/OracleCorporation.MySQLWorkbench8.0CE.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: OracleCorporation.MySQLWorkbench8.0CE
```
Comment (jedieaston) [inline manifests/o/OracleCorporation/MySQLWorkbench8/0CE/8.0.28/OracleCorporation.MySQLWorkbench8.0CE.installer.yaml:4]:
```suggestion
PackageIdentifier: Oracle.MySQLWorkbench
```

File: manifests/o/OracleCorporation/MySQLWorkbench8/0CE/8.0.28/OracleCorporation.MySQLWorkbench8.0CE.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: OracleCorporation.MySQLWorkbench8.0CE
```
Comment (jedieaston) [inline manifests/o/OracleCorporation/MySQLWorkbench8/0CE/8.0.28/OracleCorporation.MySQLWorkbench8.0CE.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Oracle.MySQLWorkbench
```

File: manifests/o/OracleCorporation/MySQLWorkbench8/0CE/8.0.28/OracleCorporation.MySQLWorkbench8.0CE.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: OracleCorporation.MySQLWorkbench8.0CE
```
Comment (jedieaston) [inline manifests/o/OracleCorporation/MySQLWorkbench8/0CE/8.0.28/OracleCorporation.MySQLWorkbench8.0CE.yaml:4]:
```suggestion
PackageIdentifier: Oracle.MySQLWorkbench
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Thanks for the submission! MySQL on Windows can be a pain, so if you are still having issues please don't hesitate to ask questions. We've had trouble with it before :)

Also, once you accept these changes, you'll need to change the folder names. A quick way to do that is by using the web version of VSCode built into GitHub:

Press "." on your keyboard when you have GitHub open to this PR in your browser, and a web version of VSCode will launch.
 
 Once it loads, find the files you added, and rename the folder (You'll need to move these files to `o\Oracle\MySQLWorkbench`). Then use the source control button on the left to commit (it's automatically pushed).
 
 image

**Comment by jedieaston** [comment]
Dependency support isn't in the client used on the test runner yet, you might have to wait on this until dependencies are in stable.

**Comment by filipponova** [comment]
@jedieaston, hello. I adjusted the installer file to contemplate the dependency, but even so I keep getting installation error, locally I don't see the dependency installation either, I tested it in local Windows Librarie mode, but without success. Can you help me?

**Comment by filipponova** [comment]
> Dependency support isn't in the client used on the test runner yet, you might have to wait on this until dependencies are in stable.

Got it, okay. I will wait for the next steps.

**Comment by jedieaston** [comment]
Dependency support is currently only available in the preview release of winget: https://github.com/microsoft/winget-cli/releases/tag/v1.2.3411-preview

You can test it locally by installing that and then adjusting your `winget settings`: https://github.com/microsoft/winget-cli/blob/master/doc/Settings.md#dependencies

Since this build isn't being used on the build runners, it still won't validate, but you'll know that you did it right at least. Since dependency support should be in the next release, you shouldn't need to wait too much longer...

**Comment by filipponova** [comment]
> Atualmente, o suporte a dependências está disponível apenas na versão prévia do winget: [https://github.com/microsoft/winget-cli/releases/tag/v1.2.3411-preview ](https://github.com/microsoft/winget-cli/releases/tag/v1.2.3411-preview)
> 
> Você pode testá-lo localmente instalando-o e ajustando seu `winget settings`: [https://github.com/microsoft/winget-cli/blob/master/doc/Settings.md#dependencies ](https://github.com/microsoft/winget-cli/blob/master/doc/Settings.md#dependencies)
> 
> Como essa compilação não está sendo usada nos executores de compilação, ela ainda não será validada, mas você saberá que pelo menos fez certo. Como o suporte à dependência deve estar na próxima versão, você não precisa esperar muito mais...

Tha

**Comment by filipponova** [comment]
image
@jedieaston ,

In my local tests with version v1.2.3411-preview it worked correctly. I just needed to replace the dependency from Microsoft.VC++2015Redist-x86 to Microsoft.VC++2015-2019Redist-x64. Thanks so much for all the support.

**Comment by jedieaston** [comment]
This is blocked waiting on winget 1.2.

**Comment by jedieaston** [comment]
1.2 may not have package dependency support. So I don't know when this will be merged.

**Comment by jedieaston** [comment]
Waiting on dependencies...

**Pull request closed without merging**